README.md

```diff
@@ -95,6 +95,6 @@ repeat these steps to upgrade to future releases.
 ## Building
 
 * [Linux](docs/build-instructions/linux.md)
-* [OS X](docs/build-instructions/os-x.md)
+* [macOS](docs/build-instructions/macos.md)
```

Comment:
Please don't change paths for branding purposes without having redirects, this isn't worth breaking links over